modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -751,40 +751,40 @@
 
 // Donation reward for CandleJax
 /obj/item/clothing/head/helmet/space/plasmaman/candlejax
-	name = "emission's helmet"
-	desc = "A special containment helmet designed for heavy usage. Multiple dings and notches are on this one."
+	name = "Specialised helmet"
+	desc = "A specialised helmet designed for heavy usage. Designed for Plasmamen who fancy a new look."
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
 	icon_state = "emissionhelm"
 
 
 // Donation reward for CandleJax
 /obj/item/clothing/head/helmet/space/plasmaman/candlejax2
-	name = "azulean's environment helmet"
-	desc = "An Azulean-made Enviro-Helmet, adjusted for the unique skull shape typical of the species. Alongside the standard features, it includes an embossment of the Azulean Crest on the back of the helmet."
+	name = "Skullshape environment helmet"
+	desc = "A new Enviro-Helmet, adjusted for the unique skull shape typical of some Plasmamen."
```

Comment:
spelling error
description doesnt make sense ("the typical shape of some plasmamen" what?)